### PR TITLE
Modified test list for papi/syspapi samplers & removed rapl

### PR DESCRIPTION
**Pull request by Snell1224**
1. In reference to pull request #527 , the ldms-static-test-list has been updated to correctly check the syspapi and papi sampler options. In configure.ac, the "OPTION_DEFAULT_DISABLE/ENABLE" that was originally used for the syspapi/papi samplers was changed to "HAVE_LIBPAPI" and "HAVE_LIBPFM".

2. Removed rapl in ldms-static-test-list. The check for this was removed in configure.ac and from what I understand, it is an experimental/retired sampler and don't think it will hurt to remove it from the make "check/installcheck" testing.

3. Updated the ldms-run-static-tests man page to the correct script type (.sh to .test).

### CODE REVIEW

File: ldms/src/test/ldms-static-test-list.sh.in [hunk 1]
```diff
@@ -3,15 +3,14 @@
 #check which plugins are enabled and store into an array.
 @ENABLE_MEMINFO_TRUE@ testlist[${#testlist[@]}]="meminfo"
 @ENABLE_APP_SAMPLER_TRUE@ testlist[${#testlist[@]}]="app_sampler"
-@ENABLE_PAPI_SAMPLER_TRUE@ testlist[${#testlist[@]}]="papi_sampler"
-@ENABLE_SYSPAPI_SAMPLER_TRUE@ testlist[${#testlist[@]}]="syspapi_sampler"
+@HAVE_LIBPAPI_TRUE@ testlist[${#testlist[@]}]="papi_sampler"
+@HAVE_LIBPFM_TRUE@ testlist[${#testlist[@]}]="syspapi_sampler"
```
Comment (morrone) [inline ldms/src/test/ldms-static-test-list.sh.in:7]: This should really be:

@HAVE_LIBPAPI_TRUE@@HAVE_LIBPFM_TRUE@

It depends on both.

Comment (Snell1224) [reply ldms/src/test/ldms-static-test-list.sh.in:7]: Okay I see my mistake, will make the necessary changes. Does the syspapi sampler also depend on both?

Thanks for letting me know.

Comment (morrone) [reply ldms/src/test/ldms-static-test-list.sh.in:7]: syspapi_sampler is the one that needs both, papi_sampler does not.

Comment (Snell1224) [reply ldms/src/test/ldms-static-test-list.sh.in]: Okay got it. Updated the script.

**Pull request merged**